templates/components/multiagent/python/app/api/routers/vercel_response.py

```diff
@@ -55,32 +54,26 @@ def _create_stream(
         self,
         request: Request,
         chat_data: ChatData,
-        event_handler: Awaitable[AgentRunResult | AsyncGenerator],
-        events: AsyncGenerator[AgentRunEvent, None],
+        event_handler: Awaitable,
```

Comment:
why remove the type hints?

Reply:
we have a type checking for the result of the event handler already. I removed the hint to not import anything from single.py code in the vercel response.